src/createServer.js

```diff
@@ -1,11 +1,37 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { userRouter } = require('./routes/users.route');
+const { expensesRouter } = require('./routes/expenses.route');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  /* CORS  */
+  const options = {
+    origin: '*',
+    methods: 'GET, POST, PUT, DELETE , PATCH',
+    allowedHeaders: 'Content-Type',
+    credentials: true,
+  };
+
+  app.use(cors(options), express.json());
+
+  /* ROUTES */
+  app.use('/users', userRouter);
+  app.use('/expenses', expensesRouter);
+
+  /* ERRORS */
+  app.use((_, res) => {
+    res.status(404).json({ message: 'Not found' });
+  });
```

Comment:
The 404 error handler is placed before the general error handler. This is correct, but ensure that no other middleware or routes are added after this point, as they would not be reached.